### DISCUSSION TITLE
Vs funk mod on android port by me

### DISCUSSION THREAD

**Discussion by StrYk85** [luckydog7/Funkin-android#1613] [mods]
WASSUP IM BACK BIT-
ok here is the apk:https://drive.google.com/file/d/1LklnYiG8raAvbs9VzefECl30eRZney_P/view?usp=drivesdk

Also funk is floating,i tried to edit his xml,but im too lazy to continue remaking the xml
And if you select the third song it will crash,because there is no 3rd song in the Vs funk mod

Here are some screenshots
![Screenshot_20210706-221317](https://user-images.githubusercontent.com/79270509/124655399-75f82200-dea8-11eb-9013-1b81f93a63e4.jpg)
![Screenshot_20210706-221323](https://user-images.githubusercontent.com/79270509/124655408-798ba900-dea8-11eb-8e70-43a58b5a661a.jpg)
![Screenshot_20210706-221331](https://user-images.githubusercontent.com/79270509/124655420-7c869980-dea8-11eb-9952-c5561adfa99a.jpg)
![Screenshot_20210706-221344](https://user-images.githubusercontent.com/79270509/124655432-801a2080-dea8-11eb-88fa-88f0eff02176.jpg)
![Screenshot_20210706-221351](https://user-images.githubusercontent.com/79270509/124655439-827c7a80-dea8-11eb-973c-1749a068422a.jpg)
![Screenshot_20210706-221409](https://user-images.githubusercontent.com/79270509/124655445-84ded480-dea8-11eb-9643-4d5cae6dd041.jpg)

Also I added dialogue
The custom week template is by Iprimegamerz
His channel:https://m.youtube.com/channel/UCpWyByigLOl3Cnzux157CHw
My channel:https://www.youtube.com/channel/UCH3Cg6ks7yErGWtMiISiOog

**Comment by Alessio2010** [comment]
Whoooo

**Comment by Alessio2010** [comment]
Iiiiiis heeeeeee

**Comment by Alessio2010** [comment]
But ok
